client/main/backstage.js

```diff
@@ -442,10 +446,12 @@ const FSEntryBrowserView = SilkyView.extend({
                             isExample: false,
                             tags: ['Online data set'],
                             description: description,
+                            isUrl: true,
+                            skipExtensionCheck: true,
                         }];
-                        isUrl = true;
+
                     } catch (e) {
-                        isUrl = false;
```

Comment:
this catch was part of the process of determining the validity of the url using the exceptions that the URL class would throw. Is there a reason that this was removed?

Reply:
i guess the item stores the isurl now.

Reply:
if an exception is thrown, then `isUrl` will never be set to true.